docs/UserGuide.md

```diff
@@ -337,16 +337,20 @@ App data are saved automatically as a JSON file `[JAR file location]/data/addres
 <box type="warning" seamless>
 
 **Caution:**
-If your changes to the data file makes its format invalid, app will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
-Furthermore, certain edits can cause the app to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+If your changes to the data file makes its format invalid, app will discard all data at the start of the next run. <br>
+An empty data file will replace the corrupted file once a new successful command is executed.
```

Comment:
Technically, the data file is not empty if the successful command is an add command. Would it be better to say "A new data file will replace ..."

Reply:
ok makes sense